charts/ai-service/templates/deployment.yaml

```diff
@@ -188,4 +154,20 @@ spec:
               secretProviderClass: {{ $fullName }}-{{ $k }}
       {{- end }}
       {{- end }}
+      {{- if .Values.artifactsCache.enabled }}
+      initContainers:
+        - name: download-artifacts
+          image: python:3.9-slim
+          command:
+            - /scripts/download-artifacts.sh
```

Comment:
This part needs to be dynamic depending on the service that is calling it.

I think perhaps, what we want to add in the sample for the values, is the `content of command` or `args`(depending on how we download design the function in the .sh), `name`, vaolumeMounts-name, and that's it.